src/app/window-handler.ts

```diff
@@ -562,6 +562,9 @@ export class WindowHandler {
     );
 
     this.mainWebContents.on('did-finish-load', async () => {
+      await this.mainWebContents?.executeJavaScript(
+        `window.addEventListener("mouseup", (e)=>{if (e.button === 3 || e.button === 4) {console.log("Mouse " + e.button + " clicked.");e.stopPropagation();e.preventDefault()}})`,
```

Comment:
@NguyenTranHoangSym Can we not directly write this in `ssf-api.ts` like we already do for other events? https://github.com/finos/SymphonyElectron/blob/main/src/renderer/ssf-api.ts#L1317-L1322

Reply:
Yup. Already tested but remaining one case unexpected. In the concept of navigation we had the loading frame in which API is pended calling to endpoint and awaiting for response. Sadly I cannot handle this case so I may comment it as an exception. Electron did not seem to provide any interception method